services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -249,7 +249,7 @@ async def patch_project(
     *,
     user_id: UserID,
     project_uuid: ProjectID,
-    project_patch: ProjectPatch,
+    project_patch: ProjectPatch | ProjectPatchExtended,
```

Comment:
why the `or` here? the trash has a default value. and it works down the line cause it gets converted to a dict I see.

Reply:
This is linked to https://github.com/ITISFoundation/osparc-simcore/issues/6595

The problem is that at the "service"layer (I am referring at the "controller-service-repository" layer) we want to use an extended version of the patch model than in the "controller" layer.  In this case `ProjectPatch` is a model from the "controller" layer and "ProjectPatchExtended" (which inherits from the former) is a model of the "service" layer